1-exercises/C-while-loop-with-array/exercise.js

```diff
@@ -18,6 +18,13 @@ const BIRTHDAYS = [
 
 function findFirstJulyBDay(birthdays) {
     // TODO
+    let i = 0; 
+    while (i < birthdays.length) {
+        if (birthdays[i].includes("July")) {
+            return birthdays[i];
+         }
+        i++;
+     }
 }
 
 console.log(findFirstJulyBDay(BIRTHDAYS)); // should output "July 11th"
```

Comment:
refer to https://syllabus.codeyourfuture.io/guides/code-style-guide#dont-leave-lots-of-commented-out-code